GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -226,7 +226,7 @@ private ContextMenuSelectionInfo GetSelectionInfo()
             var isCombinedDiff = isExactlyOneItemSelected && DiffFiles.CombinedDiff.Text == DiffFiles.SelectedItemParent;
             var selectedItemStatus = DiffFiles.SelectedItem;
             bool isBareRepository = Module.IsBareRepository();
-            bool singleFileExists = isExactlyOneItemSelected && File.Exists(DiffFiles.SelectedItem.Name); 
+            bool singleFileExists = isExactlyOneItemSelected && File.Exists(FormBrowseUtil.GetFullPathFromGitItemStatus(Module, DiffFiles.SelectedItem));
```

Comment:
Similar to https://github.com/gitextensions/gitextensions/pull/4317#discussion_r160849662 I would prefer `IFullPathResolver`.

Reply:
Can we do this in a separate PR instead and apply a similar change for all similar usages?